Packs/Cyberint/Integrations/FeedCyberint/README.md

```diff
@@ -0,0 +1,290 @@
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Cyberint Feed.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | The Cyberint IOC feed endpoint URL |  | True |
+    | API Key (Leave empty. Fill in the API Key in the password field.) |  | True |
+    | API access token |  | True |
+    | Fetch indicators |  | False |
+    | Indicator Reputation | Indicators from this integration instance will be marked with this reputation | False |
+    | Source Reliability | Reliability of the source providing the intelligence data | True |
+    | Traffic Light Protocol Color | The Traffic Light Protocol \(TLP\) designation to apply to indicators fetched from the feed | False |
+    | Indicator Type | Which indicator types to fetch | True |
+    | Feed Name |  | True |
+    | Confidence | Confidence about the indicator details. The value of confidence to fetch indicators from. The value between 0-100. | False |
+    | Severity | Severity about the indicator details. The value of severity to fetch indicators from. The value between 0-100. | False |
+    |  |  | False |
+    |  |  | False |
+    | Feed Fetch Interval |  | False |
+    | Tags | Supports CSV values. | False |
+    | Bypass exclusion list | When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system. | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### cyberint-get-indicators
+
+***
+Gets indicators from the feed.
+
+#### Base Command
+
+`cyberint-get-indicators`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | The maximum number of results to return. The default value is 10. Default is 10. | Optional |
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Cyberint.value | String | The indicator value. |
+| Cyberint.type | String | The indicator type. |
+| Cyberint.Tags | String | Tags that are associated with the indicator. |
+| Cyberint.description | String | The feed description. |
+| Cyberint.detected_activity | String | The feed detected activity. |
+| Cyberint.observation_date | String | The feed observation date. |
+| Cyberint.severity_score | String | The feed severity score. |
+| Cyberint.confidence | String | The feed confidence. |
+
+#### Command example
+```!cyberint-get-indicators limit=10 execution-timeout=700```
```

Comment:
```suggestion
```!cyberint-get-indicators limit=10```
```

Reply:
The command will return a time-out error when not using the `execution-timeout` argument.